modules/fx/arrow-fx-reactor/src/test/kotlin/arrow/fx/FluxKTest.kt

```diff
@@ -61,13 +66,25 @@ class FluxKTest : UnitSpec() {
       }
   }
 
+  fun GENK() = object : GenK<ForFluxK> {
+    override fun <A> genK(gen: Gen<A>): Gen<Kind<ForFluxK, A>> =
+      Gen.list(gen).map { Flux.fromIterable(it).k() }
+  }
+
+  fun EQK(): EqK<ForFluxK> = object : EqK<ForFluxK> {
+    override fun <A> Kind<ForFluxK, A>.eqK(other: Kind<ForFluxK, A>, EQ: Eq<A>): Boolean =
+      EQ<A>().run {
+        this@eqK.fix<A>().eqv(other.fix())
+      }
+  }
```

Comment:
@pakoito I don't know enough about the reactor/rx2 etc instances to check if these generators/eq instances are good. Can you take a look over them?

Also this suffers from the same problem the `Free` gen has: There are no easy ways to define recursive gens, otherwise we could have actual programs to test.